src/gnatss/utilities/time.py

```diff
@@ -2,11 +2,14 @@
 
 Time utilities module utilizing astropy
 """
+from typing import Union
+
+from astropy import units as u
 from astropy.time import Time as AstroTime  # noqa
+from astropy.time import TimeDelta
 from astropy.time.formats import TimeFromEpoch, erfa
-from astropy.units import isclose
```

Comment:
I thought we are using the `utilities/time.py` as an interface to expose only required Astropy functionalities. So it made sense to expose `atropy.units.isclose()` as a public API from this utility file.
(n the whole project, this is the only file where Astropy library is imported)

Reply:
This function is used only in the tests so we don't need it to be exposed for users.